gems/pending/util/xml/miq_rexml.rb

```diff
@@ -106,6 +105,7 @@ def self.findElementInt(paths, ele)
       return ele
     end
   end
+  private_class_method :findElementInt
```

Comment:
Unfortunately, I think we have to leave this as it was...   Maybe just drop this change or add a comment.  We really have to dump all this rexml stuff, but not today sadly.  😢   cc @gmcculloug 

I can't tell if this is being called with a receiver:

```
git grep findElementInt
gems/pending/util/xml/miq_rexml.rb:    findElementInt(path.tr("\\", "/").split("/"), element)
gems/pending/util/xml/miq_rexml.rb:  def self.findElementInt(paths, ele)
gems/pending/util/xml/miq_rexml.rb:          return findElementInt(paths, e)
gems/pending/util/xml/miq_rexml.rb:  private_class_method :findElementInt
gems/pending/util/xml/xml_hash.rb:    def self.findElementInt(paths, ele)
gems/pending/util/xml/xml_hash.rb:            return findElementInt(paths, e)
gems/pending/util/xml/xml_utils.rb:    return XmlHash::XmhHelpers.findElementInt(path_fix_up, element) if element.kind_of?(Hash)
gems/pending/util/xml/xml_utils.rb:    findElementInt(path_fix_up, element)
gems/pending/util/xml/xml_utils.rb:  def self.findElementInt(paths, ele)
gems/pending/util/xml/xml_utils.rb:          return findElementInt(paths, e)
```